en/mkdocs.yml

```diff
@@ -69,6 +69,7 @@ nav:
           - Ballerina Expression Syntax: references/ballerina-expression-syntax.md
           - Anonymous Access: references/anonymous-access.md
           - Data Mapping: references/data-mapping.md
+          - Postman Integration: references/postman-integration.md
```

Comment:
Need to change this also to reflect the title change.

Reply:
Changed navigator title